src/app/modules/group/pages/group-user/group-user.component.ts

```diff
@@ -1,24 +1,80 @@
-import { Component } from '@angular/core';
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
 import { User, GetUserService } from '../../http-services/get-user.service';
 import { mapToFetchState } from '../../../../shared/operators/state';
-import { Observable } from 'rxjs';
-import { ActivatedRoute } from '@angular/router';
-import { switchMap } from 'rxjs/operators';
+import { Observable, Subscription } from 'rxjs';
+import { ActivatedRoute, NavigationEnd, Router, RouterLinkActive } from '@angular/router';
+import { delay, switchMap, map, filter, startWith } from 'rxjs/operators';
+import { contentInfo } from '../../../../shared/models/content/content-info';
+import { CurrentContentService } from '../../../../shared/services/current-content.service';
+import { UserSessionService } from '../../../../shared/services/user-session.service';
 
 @Component({
   selector: 'alg-group-user',
   templateUrl: './group-user.component.html',
   styleUrls: [ './group-user.component.scss' ]
 })
-export class GroupUserComponent {
+export class GroupUserComponent implements OnInit, OnDestroy {
+  @ViewChild('progress') progress?: RouterLinkActive;
+  @ViewChild('personalData') personalData?: RouterLinkActive;
+
   readonly state$ = this.route.params.pipe(
     switchMap(({ id }) => this.getUser$(id)),
-    mapToFetchState()
+    mapToFetchState(),
+  );
+
+  readonly currentUserGroupId$ = this.userSessionService.userProfile$.pipe(
+    delay(0),
```

Comment:
Have you experienced that this `delay(0)` was required?

Reply:
I have found this in our app, I suppose async pipe, in template, doesn't work under *ngIf, delay(0) (which is async) moves this operation to end of stack operations and makes it work under *ngIf.

Reply:
So this is just a trick